index.bs

```diff
@@ -1741,7 +1741,7 @@ these steps:
   1. Perform the <a spec=html>activation notification</a> steps in |global|'s [=Window/browsing context=].
 
   1. [=Request permission to use=] |desc|.
-  1. Run the [=boolean permission query algorithm=] on |desc| and |status|.
+  1. Run the <a spec=permissions>default permission query algorithm</a> on |desc| and |status|.
```

Comment:
@mkruisselbrink this instance was not updated in https://github.com/WICG/file-system-access/commit/dc8b56dc01ee3ad8bbb63ecd804b5be79c13262a. I assume that was not intentional?

Reply:
I can't imagine it was, no.